src/client/app/translations/data.ts

```diff
@@ -846,84 +846,86 @@ const LocaleTranslationData = {
+		"meter.readingGap": "Reading Gap:",
+		"meter.readingVariation": "Reading Variation:",
+		"meter.startTimeStamp": "Start Time Stamp:",
+		"meter.successfully.create.meter": "Successfully created a meter.",
+		"meter.successfully.edited.meter": "Successfully edited meter.",
+		"meter.timeSort": "Time Sort:",
 		"meter.time.zone": "fuseau horaire du mètre",
 		"meter.type": "Type de Mèters",
-		"minute": "Minute\u{26A1}",
-		"min": "min\u{26A1}",
+		"minute": "Minute",
+		"min": "min",
+		"misc": "Divers",
 		"more.energy": "plus d'énergie",
 		"name": "Nom:",
 		"navigation": "Navigation",
-		"need.more.points": "Need more points\u{26A1}",
-		"no": "no\u{26A1}",
+		"need.more.points": "Need more points",
+		"no": "no",
 		"note": "Noter:",
-		"no.data.in.range": "No Data In Date Range\u{26A1}",
+		"no.data.in.range": "No Data In Date Range",
 		"oed": "Tableau de Bord Ouvert d'énergie",
 		"oed.description": "Le Tableau de Bord Ouvert d'énergie est un projet open source indépendant. ",
-		"oed.version": "OED version \u{26A1}",
+		"oed.version": "OED version ",
 		"options": "Options",
-		"page.choice.login": "Page choices & login\u{26A1}",
-		"page.choice.logout": "Page choices & logout\u{26A1}",
+		"page.choice.login": "Page choices & login",
+		"page.choice.logout": "Page choices & logout",
 		"password": "Mot de passe",
-		"password.confirm": "Confirm password\u{26A1}",
-		"per.day": "Per Day\u{26A1}",
-		"per.hour": "Per Hour\u{26A1}",
-		"per.minute": "Per Minute\u{26A1}",
-		"per.second": "Per Second\u{26A1}",
+		"password.confirm": "Confirm password",
+		"per.day": "Per Day",
+		"per.hour": "Per Hour",
+		"per.minute": "Per Minute",
+		"per.second": "Per Second",
 		"projected.to.be.used": "projeté pour être utilisé",
 		"radar": "Radar",
-		"radar.lines.incompatible": "These meters/groups are not compatible for radar graphs\u{26A1}",
-		"radar.no.data": "There are no readings:<br>likely the data range is outside meter/group reading range\u{26A1}",
-		"rate": "Rate\u{26A1}",
-		"reading": "Reading:\u{26A1}",
-		"redo.cik.and.refresh.db.views": "Processing changes. This may take a while\u{26A1}",
-		"readings.per.day": "Readings per Day\u{26A1}",
+		"radar.lines.incompatible": "These meters/groups are not compatible for radar graphs",
+		"radar.no.data": "There are no readings:<br>likely the data range is outside meter/group reading range",
+		"rate": "Rate",
+		"reading": "Reading:",
+		"redo.cik.and.refresh.db.views": "Processing changes. This may take a while",
+		"readings.per.day": "Readings per Day",
 		"redraw": "Redessiner",
-		"remove": "Remove\u{26A1}",
+		"remove": "Remove",
 		"restore": "Restaurer",
-		"return.dashboard": "Return To Dashboard\u{26A1}",
-		"role": "Role\u{26A1}",
-		"save.all": "Save all\u{26A1}",
-		"save.map.edits": "Save map edits\u{26A1}",
+		"return.dashboard": "Return To Dashboard",
+		"role": "Rôle",
+		"save.all": "Sauver tous",
+		"save.map.edits": "Enregistrer les modifications de la carte",
```

Comment:
You seem to have translated strings that are not directly involved in this PR. Is that correct or am I missing something? If so then: First, should this be a separate PR? Second, since you only did a few, is this a language you are very good at so the translations will be of high quality?